components/HotWallet/plugins/TribecaGauges/EscrowData.tsx

```diff
@@ -0,0 +1,83 @@
+import { BN } from '@project-serum/anchor'
+import type { EscrowData } from '@tools/sdk/tribeca/programs'
+import { tryGetTokenMint } from '@utils/tokens'
+import BigNumber from 'bignumber.js'
+import moment from 'moment'
```

Comment:
Looks unused too ?
If we need to manipulate dates & durations & stuff, there's a high chance native DOM APIs can already do the job, if not, there are plently of alternatives such as dayjs, date-fns, luxon which are much better than moment.

Reply:
Changed it to use native Date instead of moment. You are right it's better :) 
  
  `return new Date(dateInSec.mul(new BN(1000)).toNumber()).toUTCString()`